libraries/HTTPClient/src/HTTPClient.cpp

```diff
@@ -495,7 +495,7 @@ void HTTPClient::setTimeout(uint16_t timeout)
 {
     _tcpTimeout = timeout;
     if(connected()) {
-        _client->setTimeout((timeout + 500) / 1000);
+        _client->setTimeout(timeout + 500); /* / 1000 removed, WifiClient setTimeout changed to ms */
```

Comment:
@P-R-O-C-H-Y I would suggest to put notes such as this into commit message body, instead of adding them as comments. The reader of the code is probably not interested in every change which has been done to each line, they just need to understand the final version.

For example, I don't understand why there is timeout+500 here.

I think previously +500 in (timeout + 500)/1000 was related to rounding. But with /1000 removed, looks like +500 is not needed anymore?

Reply:
Yes, this was there for rounding. `+500` should be removed

Reply:
The rounding was removed, missed that.
Thanks for suggestions about commenting the code / commits. Will have it in mind for future changes :)